java-core/google-cloud-core/src/main/java/com/google/cloud/ServiceOptions.java

```diff
@@ -862,6 +862,11 @@ public String getResolvedHost(String serviceName) {
    */
   @InternalApi
   public String getResolvedApiaryHost(String serviceName) {
+    // The host value set to DEFAULT_HOST if the user didn't configure a host. If the
+    // user set a host the library uses that value, otherwise, construct the host for the user.
```

Comment:
These should be in Javadoc.

Reply:
And the javadoc shouldn't start with "Temporarily used for ...". It starts with "Returns ..."

Reply:
Added.